src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/GeneralPartnerServiceTest.java

```diff
@@ -74,6 +78,14 @@ void testCreateLinksForGeneralPartnerReturnsSuccess() throws ServiceException {
         assertEquals(expectedUri, sentSubmission.getLinks().get("self"));
     }
 
+    @Test
+    void testExceptionIsThrownWhenFirstAndSecondNationalityIsTheSame() throws ServiceException, MethodArgumentNotValidException, NoSuchMethodException {
```

Comment:
the throws are not needed on the test signature

Reply:
Removed

Reply:
Please see [this](https://github.com/companieshouse/limited-partnerships-api/pull/83#issuecomment-2684370245) comment.